libraries/AP_Mount/AP_Mount.h

```diff
@@ -106,6 +106,10 @@ class AP_Mount
     enum MAV_MOUNT_MODE get_mode() const { return get_mode(_primary); }
     enum MAV_MOUNT_MODE get_mode(uint8_t instance) const;
 
+    // get_rc_invalid_mode - returns rc invalid pointing coordinates of mount (i.e. no change, Retracted, Neutral)
```

Comment:
Let's eliminate these getters.  They're just noise in the interface.

Reply:
scope creep? not intent of this PR to correct past code mistakes, right? I am barely able to get the intended operation to work ;)

Reply:
done